libs/ts-rest/nest/src/lib/ts-rest-nest.spec.ts

```diff
@@ -22,7 +23,12 @@ it('allows unknown statuses when not in strict mode', () => {
   const cLoose = c.router({ posts: postsRouter });
   const nestContract = nestControllerContract(cLoose.posts);
   type RequestShapes = NestRequestShapes<typeof nestContract>;
+  type ResponseShapes = NestResponseShapes<typeof nestContract>;
 
+  const responseTypeCheck: Awaited<ResponseShapes['getPost']> = {
```

Comment:
Would be nicer to use `Expect<Equal<>>` here but I don't think it is in scope as it is in the core library?